proposals/NNNN-SDL-behavior-in-case of-Low-Voltage.md

```diff
@@ -0,0 +1,74 @@
+# SDL behavior in case of LOW_VOLTAGE event
+
+* Proposal: [SDL-NNNN](NNNN-filename.md)
+* Author: [Alexander Kutsan](https://github.com/LuxoftAKutsan)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / Web / RPC / Protocol]
+
+## Introduction
+
+This proposal defines SDL behavior in case of LOW_VOLTAGE event.
+LOW_VOLTAGE scenario is triggered when the battery voltage hits below 7v. In case of such event EMMC is turned off and all read/write operations are unavailable. After the voltage level is restored all oprations are resumed.
```

Comment:
@dboltovskyi please remove hardcoded 7v value

Reply:
Done.

Reply:
@AStasiuk , I still see the reference to 7v